apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -12,18 +12,19 @@ defmodule Aecore.Miner.Worker do
   alias Aecore.Structures.Header
   alias Aecore.Structures.Block
   alias Aecore.Pow.Cuckoo
-  alias Aecore.Keys.Worker, as: Keys
   alias Aecore.Structures.TxData
   alias Aecore.Structures.SignedTx
   alias Aecore.Chain.ChainState
   alias Aecore.Txs.Pool.Worker, as: Pool
   alias Aeutil.Bits
   alias Aecore.Peers.Worker, as: Peers
+  alias Aecore.Wallet.Worker, as: Wallet
 
   require Logger
 
   @mersenne_prime 2147483647
   @coinbase_transaction_value 100
+  @wallet_default_pass "1234"
```

Comment:
why do we choose "1234" as password over "" as default?

Reply:
Hmm I just did thought of that, I'll change it